README.md

```diff
@@ -1,45 +1,221 @@
-# DKG.py
+<a name="readme-top"></a>
+
+> **Disclaimer: Beta Version**
```

Comment:
In my opinion this disclaimer should be smaller, no need for so much explanation also we don't plan to keep this forever